compose/ui/ui/src/uikitInstrumentedTest/kotlin/android/compose/ui/test/UIKitInstrumentedTest.kt

```diff
@@ -81,7 +81,11 @@ internal class UIKitInstrumentedTest {
     ) {
         var onDraw = false
```

Comment:
> iOS - Features

Please, change it to Breaking Changes - iOS

(moved from https://github.com/JetBrains/compose-multiplatform-core/pull/1451#issuecomment-2230485628)

Reply:
Done